codemeta.json

```diff
@@ -1,703 +1,372 @@
+            "@id": "https://orcid.org/0000-0003-0426-6634"
         },
         {
-            "@id": "https://orcid.org/0000-0002-3134-1946",
             "@type": "Person",
-            "affiliation": {
-                "@type": "Organization",
-                "name": "Max Planck Institut f\u00fcr Kernphysik, MPIK"
-            },
-            "familyName": "Chakraborty",
-            "givenName": "Nachiketa"
+            "givenName": "David",
+            "familyName": "Carreto Fidalgo",
+            "affiliation": "Max Planck Computing and Data Facility, Gie\u00dfenbachstra\u00dfe 2, 85748 Garching, Germany",
+            "@id": "https://orcid.org/0000-0002-7480-2730"
         },
         {
             "@type": "Person",
-            "affiliation": {
-                "@type": "Organization",
-                "name": "Department of Physics, University of the Free State"
-            },
-            "familyName": "W. Chen",
-            "givenName": "Andrew"
+            "givenName": "Andrew",
+            "familyName": "Chen",
+            "affiliation": "University of the Witwatersrand, South Africa",
+            "@id": "https://orcid.org/0000-0001-6425-5692"
         },
         {
             "@type": "Person",
-            "affiliation": {
-                "@type": "Organization",
-                "name": "Iowa State University"
-            },
+            "givenName": "Alisha",
             "familyName": "Chromey",
-            "givenName": "Alisha"
+            "affiliation": "Iowa State University, USA"
         },
         {
-            "@id": "https://orcid.org/0000-0001-7282-2394",
             "@type": "Person",
-            "affiliation": {
-                "@type": "Organization",
-                "name": "GAE-UCM"
-            },
-            "familyName": "Contreras",
-            "givenName": "Jos\u00e9 Luis"
+            "givenName": "Jose Luis",
+            "familyName": "Contreras Gonzalez",
+            "affiliation": "IPARCOS Institute and EMFTEL Department, Universidad Complutense de Madrid, Spain",
+            "@id": "https://orcid.org/0000-0001-7282-2394"
         },
         {
             "@type": "Person",
-            "affiliation": {
-                "@type": "Organization",
-                "name": "Universidade de S\u00e3o Paulo, Brazil"
-            },
-            "familyName": "de Almeida Coutinho",
-            "givenName": "Alexis"
+            "givenName": "Mathieu",
+            "familyName": "de Bony de Lavergne",
+            "affiliation": "Laboratoire d'Annecy de Physique des Particules, France"
         },
         {
-            "@id": "https://orcid.org/0000-0002-4198-4005",
             "@type": "Person",
-            "affiliation": {
-                "@type": "Organization",
-                "name": "HeidelbergCement"
-            },
-            "familyName": "Deil",
-            "givenName": "Christoph"
+            "givenName": "Jos\u00e9 Vin\u00edcius",
```

Comment:
Are those characters expected?

Reply:
UTF8 and Jason... To be honest, I do not know.